frequently-encountered-problems.html

```diff
@@ -75,6 +75,17 @@ <h3>Tedious Configurations</h3>
 
     </dd>
     <br />
+    <dt>
+        <h2>Providing a client ID when using <code>azure-active-directory-password</code></h2>
+    </dt>
+    <dd>
+        <p>The clientId field is now mandatory when using the <code>azure-active-directory-password</code> 
+            authentication type since the provided default client ID has been removed from Tedious. 
+            That default client ID will expire in the near future and stop working.</p>
```

Comment:
can we add a quick explaination why this is removed? something like "for other Microsoft owned first party driver, a hard coded client id is provided within the driver. Since tedious is not a Microsoft owner first party driver, we have to remove the hard coded client id and let user provide their own. "